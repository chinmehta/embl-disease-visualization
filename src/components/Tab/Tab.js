import React from "react";

function Tab(props) {
  return <>{props.children}</>;
}

export default Tab;
