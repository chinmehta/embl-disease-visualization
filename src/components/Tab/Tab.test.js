import React from "react";
import renderer from "react-test-renderer";

import Tab from "./Tab";

it("renders when no children is passed as prop", () => {
  const element = renderer.create(<Tab />).toJSON();

  expect(element).toMatchSnapshot();
});

it("renders when children is passed as prop", () => {
  const element = renderer
    .create(
      <Tab>
        <div>Test Text</div>
      </Tab>
    )
    .toJSON();

  expect(element).toMatchSnapshot();
});
