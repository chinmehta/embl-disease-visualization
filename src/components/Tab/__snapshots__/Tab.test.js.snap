// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when children is passed as prop 1`] = `
<div>
  Test Text
</div>
`;

exports[`renders when no children is passed as prop 1`] = `null`;
